public/app/features/dashboard/state/DashboardModel.ts

```diff
@@ -840,12 +841,12 @@ export class DashboardModel implements TimeModel {
   getRowHeight(rowPanel: PanelModel): number {
     if (!rowPanel.panels || rowPanel.panels.length === 0) {
       return 0;
+    } else if (rowPanel.collapsed) {
+      return 1;
```

Comment:
Okay, so afaiu the panels that will follow a repeated row which is collapsed will always only increase their y-position with 1. Would you put a comment here as well about this?